cluster/src/main/java/io/atomix/cluster/messaging/impl/ClusterMessage.java

```diff
@@ -1,171 +0,0 @@
-/*
- * Copyright 2014-present Open Networking Foundation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.atomix.cluster.messaging.impl;
-
-import com.google.common.base.MoreObjects;
-import io.atomix.cluster.NodeId;
-import io.atomix.utils.ArraySizeHashPrinter;
-
-import java.nio.ByteBuffer;
-import java.nio.charset.StandardCharsets;
-import java.util.Arrays;
-import java.util.Objects;
-
-/**
- * Base message for cluster-wide communications.
- */
-public class ClusterMessage {
```

Comment:
Seemed pointless to keep this, only thing that seemed to be in use was the payload.

Reply:
Yeah I think this class was actually left over from an old version of the ONOS API